mews-operations/availabilityBlockProcessor.md

```diff
@@ -0,0 +1,176 @@
+# Channel Manager API Operations: Process availability blocks
+
+## Process availability block
+
+[`async`] Use this operation to push availability blocks to Mews. The operation is used for add/update/cancel availability block.
+
+### Request
+
+[PlatformAddress]/api/channelManager/v1/processAvailabilityBlocks
+
+```json
+{
+    "clientToken": "[Channel manager client token]", 
```

Comment:
We should use actual examples here (just change a few characters to make sure it's not a valid token)